### DISCUSSION TITLE
Better sort for offline channels in sidebar

### DISCUSSION THREAD

**Discussion by CouleeApps** [night/betterttv#5952] [Ideas]
For some reason, Twitch likes to sort my subs at the bottom of the offline followed channels in the sidebar. It would be nice to have BTTV sort the list better and put the subs on top. It could also be nice to sort by most recently watched, although I'm not sure how feasible that is with the available api surface.

**Comment by night** [comment]
Thanks for the suggestion. BetterTTV changes Twitch primarily in an additive way, meaning we can add new UI (buttons, menus, emotes) or tweak existing UI through styling (hiding things, coloring things). Unfortunately this type of modification would need to completely re-render the component, and is not something we support in BetterTTV today. These changes can be brittle and can even brick the website if Twitch were to update the component.
